doc/utilities.rst

```diff
@@ -131,7 +131,7 @@ Positional Arguments
 
 .. option:: input_file(s)
 
-    One or more csv file(s), jsonlines file(s), or megam file(s) (with or without the
+    One or more csv file(s), or jsonlines file(s) (with or without the
```

Comment:
Nitpick/question: Get rid of the comma if there are only two things. But, also, why are there only two things? That is, shouldn't we mention all the file types supported here (or link to a section that enumerates them)? It seems strange to focus on CSV and jsonlines exclusively here. Also, I suggest changing "csv" and "jsonlines" to "\`\`.csv\`\`" and "\`\`.jsonlines\`\`" as they are used elsewhere (same applies to additional files added).